module/my_books.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+import sqlite3
+from telegram import Update
+from telegram.ext import CallbackContext
+from module.create_connection import create_connection
+from module.shared import DB_PATH, error_message
+from module.send_results import send_results
+
+
+def get_user_books(context: CallbackContext, chat_id: int) -> List[tuple]:
+    conn = create_connection(DB_PATH)
+    if not conn:
+        context.bot.send_message(chat_id, error_message)
+        return
+
+    user = "@" + str(context.bot.get_chat(chat_id)["username"])
+    cur = conn.cursor()
+    cur.execute("SELECT rowid, * FROM Market WHERE Venditore=?", (user,))
+    rows = cur.fetchall()
+    conn.close()
+
+    if rows:
+        context.bot.send_message(chat_id, "Hai i seguenti libri in vendita:\n")
+        send_results(rows, chat_id, context)
+    else:
+        context.bot.send_message(chat_id, "Non hai libri in vendita.")
+    return rows
```

Comment:
moreover, even here rows can be None if I understood correctly, so this can be None, would be interesting to define it in a different way, like

```python
if rows:
    ....
    return rows
 
 context.bot.send_message(...., "Non hai libri in vendita")
 ```
 
 In this way is clear that if rows is "true" so it's none you will return rows, otherwise you do not return anything
 
 not sure if this is the best and `if rows:` works only if rows is not None, may rows can return False here but it's something else
 
 feel free to keep it as it is or to remove the else and put the return inside `if rows`